nikola/plugin_categories.py

```diff
@@ -471,3 +472,221 @@ def import_file(self):
     def save_post(self):
         """Save a post to disk."""
         raise NotImplementedError()
+
+
+class Taxonomy(BasePlugin):
+    """Taxonomy for posts.
+
+    A taxonomy plugin allows to classify posts (see #2107) by
+    classification strings.
+    """
+
+    name = "dummy_taxonomy"
+
+    # Adjust the following values in your plugin!
+
+    # The classification name to be used for path handlers.
+    classification_name = "taxonomy"
```

Comment:
This class has a problem: anyone inheriting from it *will* have to adjust a lot of members, but the only way to see which ones and how is by reading source code. Even if it's more verbose, I think this help should go into the docstring.

We could also use separate docstrings, since this is parsed by most doc-extracting things:

```
foo = "bar"
"""foo is a bar thing"""
```

but it makes the source code much more confusing :-P